cmd/res/configuration.toml

```diff
@@ -2,106 +2,32 @@ MaxEventSize = 0 # value 0 unlimit the maximum event size that can be sent to me
 
 [Writable]
 LogLevel = "INFO"
-  # InsecureSecrets are required for when Redis is used for message bus
-  [Writable.InsecureSecrets]
-    [Writable.InsecureSecrets.DB]
-    path = "redisdb"
-      [Writable.InsecureSecrets.DB.Secrets]
-      username = ""
-      password = ""
   [Writable.Reading]
   ReadingUnits = true
   [Writable.Telemetry]
-  Interval = "30s"
-  PublishTopicPrefix  = "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
     [Writable.Telemetry.Metrics] # All service's metric names must be present in this list.
     # Device SDK provided Metrics
     EventsSent = false
     ReadingsSent = false
```

Comment:
This is in Device Services Common. I.e no service specific metrics.

Reply:
ok. removing them